gen/gen.go

```diff
@@ -31,7 +34,21 @@ var dirMetadata string
 // the exercise directory. Falls back to the present working directory.
 var dirExercise string
 
-// Header tells how the test data was generated, for display in the header of cases_test.go
```

Comment:
what would you say about moving the whitespace change to the formatting change commit too?